### PR TITLE
Remove OS check from PowerShell command builder

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes a part of #3477

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] ~Was the CHANGELOG.md updated to reflect the changes?~
* [ ] ~Was the documentation framework updated to reflect the changes?~
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [x] Do all unit tests pass?
* [ ] ~Do all end-to-end tests pass?~
* [ ] ~Any other testing performed?~
    > ~Tested by {Running the Monkey locally with relevant config/running Island/...}~
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

### CODE REVIEW

File: monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def execute_agent(self, agent_binary_path: PurePath, tags: Set[AgentEventTag]):
         try:
             tags.update(EXECUTION_TAGS)
             self._powershell_client.execute_cmd_as_detached_process(
-                self._command_builder(self.get_os(), agent_binary_path)
+                self._command_builder(agent_binary_path)
```
Comment (ilija-lazoroski) [inline monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py:84]: Have you tested? I think you will need to define the argument's name: `remote_agent_binary_destitination_path`.

Comment (shreyamalviya) [reply monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py:84]: Nope. Why? It worked without it previously.

Comment (ilija-lazoroski) [reply monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py:84]: It is a single argument. I know it didn't work when we did SSH.

**Review by mssalvatore — APPROVED** [review]

File: monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py (hunk 1 above)
Comment (shreyamalviya) [reply monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py:84]: EtE passes.

**Pull request merged**